Task 4/script.js

```diff
@@ -0,0 +1,30 @@
+/* ------------------------------ TASK 4 -----------------------------------
+Parašykite JS kodą, kuris vartotojui atėjus į tinklapį kreipsis į cars.json failą ir 
+atvaizduos visus automobilių gamintojus bei pagamintus modelius. 
+Kiekvienas gamintojas turės savo atvaizdavimo "kortelę", kurioje bus 
+nurodomas gamintojas ir jo pagaminti modeliai.
+
+Pastaba: Sukurta kortelė, kurioje yra informacija apie automobilį (brand), turi 
+būti stilizuota su CSS ir būti responsive;
+-------------------------------------------------------------------------- */
+
+const ENDPOINT = 'cars.json';
+
+fetch(ENDPOINT)
+  .then((response) => response.json())
+  .then((data) => {
+    let outputHtml = '';
+    data.forEach((brand) => {
```

Comment:
Šiuo atveju tai jau tampa bloga praktika, nes HTML gerokai sudėtingesnis ir viso to surašymais vien stringais didina tikimybę klaidoms ir apsunkina tolimesnį kodo palaikymą.